cms/src/components/content/ContentOverview.vue

```diff
@@ -77,12 +78,83 @@ if (!Object.entries(TagType).some((t) => t[1] == tagTypeString)) tagTypeString =
 
 const titleType = tagTypeString ? tagTypeString : props.docType;
 router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} overview`;
+
+const showFilterOptions = ref(false);
+
+const filterByTranslation = ref(queryOptions.value.translationStatus);
+const filterByTranslationOptions = [
+    {
+        value: "translated",
+        label: "Translated Documents",
+    },
+    {
+        value: "untranslated",
+        label: "Untranslated Documents",
+    },
+    {
+        value: "all",
+        label: "All Documents",
+    },
+];
+
+const filterByStatus = ref(queryOptions.value.publishStatus);
+const filterByStatusOptions = [
+    {
+        value: "published",
+        label: "Published Documents",
+    },
+    {
+        value: "scheduled",
+        label: "Scheduled Documents",
+    },
+    {
+        value: "expired",
+        label: "Expired Documents",
+    },
+    {
+        value: "draft",
+        label: "Drafted Documents",
+    },
+    {
+        value: "all",
+        label: "All Documents",
+    },
+];
```

Comment:
In my opinion we can leave out "documents". The Document concept is something we use internally to describe a piece of JSON, but for the users it might be less intuitive.